cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/Query.java

```diff
@@ -4,69 +4,112 @@
  * This class represents a query report.
  */
 public class Query {
+
+    private long startTimeInMilliseconds;
+    private long executionTimeInMilliseconds;
+    private String executionErrorMessage;
+    private String clientAddress;
+    private String statementType;
+    private String keyspace;
+    private String tableName;
+    private String statement;
+
     /**
      * Query's execution start time.
+     * @return execution start timestamp
      */
-    public long startTimeInMilliseconds;
+    public long getStartTimeInMilliseconds() {
+        return startTimeInMilliseconds;
+    }
 
     /**
      * Query's execution time.
+     * @return execution time
      */
-    public long executionTimeInMilliseconds;
+    public long getExecutionTimeInMilliseconds() {
```

Comment:
get prefix is meaningless since the class is immutable. I would prefer to leave it out.